assets/src/edit-story/components/errorBoundary/index.js

```diff
@@ -38,10 +38,7 @@ class ErrorBoundary extends Component {
 
   componentDidCatch(error, errorInfo) {
     this.setState({ error, errorInfo });
-    trackEvent('error', 'editor', '', '', {
-      error,
-      error_info: errorInfo,
-    });
+    trackError('error', 'catch error', errorInfo);
```

Comment:
```suggestion
    trackError(error, errorInfo);
```

Reply:
I am a bit confused here, will `error` be the eventCategory? And also, do we allow customized description here?

Reply:
Ah sorry, I think I jumped the gun here.

The `eventCategory` should be `editor.

So with the current function signature:

```suggestion
    trackEvent('editor', error, errorInfo, true, );
```

Reply:
As I understand `error` is the type Error, but I think we might need a eventCategory here.

Reply:
updated.